test/unit/css.js

```diff
@@ -1759,18 +1759,32 @@ QUnit.test( "Do not throw on frame elements from css method (trac-15098)", funct
 		assert.equal( elemStyle.undefined, undefined, "Nothing writes to node.style.undefined" );
 	} );
 
-	QUnit.test( "Don't detect fake set properties on a node when caching the prefixed version", function( assert ) {
-		assert.expect( 1 );
+} )();
 
-		var elem = jQuery( "<div></div>" ),
-			style = elem[ 0 ].style;
-		style.MozFakeProperty = "old value";
-		elem.css( "fakeProperty", "new value" );
+QUnit.test( "Don't detect fake set properties on a node when caching the prefixed version", function( assert ) {
```

Comment:
```suggestion
QUnit.test( "Don't set prefixed versions of fake properties", function( assert ) {
```

Since the cache is gone.

Reply:
This is not a new test, just moved outside of a block where it shouldn't be. The new one is below. There are a few more tests that mention caching.

This test specifically checks that setting a fake prefixed property doesn't affect subsequent setters. The title you proposed doesn't contain this bit. I'll think about another way to rephrase this.

Reply:
Maybe "Don't update existing unsupported prefixed properties"?